docs/core/install/linux-ubuntu.md

```diff
@@ -1,200 +1,181 @@
 
-[!INCLUDE [linux-prep-intro-apt](includes/linux-prep-intro-apt.md)]
+|| Product      | Type    | Package                  |
+|---------|--------------|---------|--------------------------|
+| **7.0**    | ASP.NET Core | Runtime | `aspnetcore-runtime-7.0` |
+| **7.0**    | .NET         | Runtime | `dotnet-runtime-7.0`     |
+| **7.0**    | .NET         | SDK     | `dotnet-sdk-7.0`         |
+| **6.0**    | ASP.NET Core | Runtime | `aspnetcore-runtime-6.0` |
+| **6.0**    | .NET         | Runtime | `dotnet-runtime-6.0`     |
+| **6.0**    | .NET         | SDK     | `dotnet-sdk-6.0`         |
 
-```bash
-wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-sudo dpkg -i packages-microsoft-prod.deb
-rm packages-microsoft-prod.deb
-```
+If you want to install an unsupported version of .NET, check the [Supported distributions](#supported-distributions) section to see if that version of .NET is available. Then, substitute the **version** of .NET you want to install. For example, to install ASP.NET Core 2.1, use the package name `aspnetcore-runtime-2.1`.
 
-[!INCLUDE [linux-apt-install-60](includes/linux-install-60-apt.md)]
+> [!TIP]
+> If you're not creating .NET apps, install the ASP.NET Core runtime as it includes the .NET runtime as well.
 
-## 16.04
+## Uninstall .NET
 
-[!INCLUDE [linux-prep-intro-apt](includes/linux-prep-intro-apt.md)]
+If you installed .NET through a package manager, uninstall in the same way with the `apt-get remove` command:
 
 ```bash
-wget https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-sudo dpkg -i packages-microsoft-prod.deb
-rm packages-microsoft-prod.deb
+sudo apt-get remove dotnet-sdk-6.0
 ```
 
-[!INCLUDE [linux-apt-install-60](includes/linux-install-60-apt.md)]
+For more information, see [Uninstall .NET](remove-runtime-sdk-versions.md?pivots=os-linux#uninstall-net).
+
+## Install preview versions
+
+[!INCLUDE [preview installs don't support package managers](./includes/linux-install-previews.md)]
 
-## How to install other versions
+## Remove preview versions
 
-[!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+[!INCLUDE [package-manager uninstall notice](./includes/linux-uninstall-preview-info.md)]
 
 ## Use APT to update .NET
 
-When a new patch release is available for .NET, you can simply upgrade it through APT with the following commands:
+If you installed .NET through a package manager, you can upgrade it with the following commands:
 
 ```bash
 sudo apt-get update
 sudo apt-get upgrade
```

Comment:
This command will update every package that has an update available and not just .NET. At a minimum, we should let the reader know this. `apt list --upgradable` will let them know what's about to be affected.